app/modules/event/src/eventSelectors.ts

```diff
@@ -33,4 +33,5 @@ export const getTotalPagesSelector = createSelector(eventSelector, events => {
 });
 
 export const getEventFilterSelector = (state: any) => state.eventsDiscovery.eventFilter;
+
```

Comment:
@AbdulmalekAlshugaa [M] extra spacing on this file , much better if you remove this unnecessary spacing.